services/net/nlp/TNO.Services.NLP.csproj

```diff
@@ -20,4 +20,9 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.7.1" />
```

Comment:
We don't need ML.NET presently, but I was playing around with it.  Seems to be more related to images than it does with text.